opdracht-tech-it-easy/src/main/java/nl/novi/opdrachttechiteasy/controllers/ExceptionController.java

```diff
@@ -0,0 +1,15 @@
+package nl.novi.opdrachttechiteasy.controllers;
+
+import nl.novi.opdrachttechiteasy.exceptions.RecordNotFoundException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+    @ControllerAdvice
+    public class ExceptionController {
+        @ExceptionHandler(RecordNotFoundException.class)
+        public ResponseEntity<String> handleRecordNotFoundException(RecordNotFoundException e) {
+            System.err.println("RecordNotFoundException occurred: " + e.getMessage());
```

Comment:
Print statements doen we in principe niet. Als je het wel graag wilt loggen, kijk dan eens of logging toe wilt passen (ook leuk voor in je verantwoordingsdocument). https://www.baeldung.com/spring-boot-logging